working-envs/activate-pytorch-dev.sh

```diff
@@ -154,6 +154,18 @@ export CFLAGS="$CFLAGS -L$CONDA_PREFIX/lib"
 export CONDA_BUILD_SYSROOT=$CONDA_PREFIX/$HOST/sysroot
 export LD_LIBRARY_PATH=$CONDA_PREFIX/lib:$LD_LIBRARY_PATH
 
+if [ -v CXXFLAGS_CONDA_NVCC_BACKUP ] ; then
+    export CXXFLAGS_CONDA_NVCC_BACKUP="`echo $CXXFLAGS_CONDA_NVCC_BACKUP | sed 's/-std=c++17/-std=c++14/'`"
+fi
```

Comment:
CXXFLAGS_CONDA_NVCC_BACKUP should not be changed, it is needed for deactivating the environment.